<launch>

  <arg name="rviz" default="true" />
  
  <arg name="params_file" default="$(find dogm_ros)/config/params.yaml"/>

  <node pkg="dogm_ros" type="dogm_node" name="dogm_node" output="screen">
    <rosparam file="$(arg params_file)"/>
  </node>

  <node pkg="dogm_ros" type="fake_scan.py" name="fake_scan" output="screen"/>

  <node pkg="dogm_ros" type="static_tf.py" name="static_tf_publisher" />

  <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz" 
        args="-d $(find dogm_ros)/rviz/dogm.rviz" />
          
</launch>